core/client/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -91,6 +91,7 @@
    */
   AbstractFileSystem(FileSystem fileSystem) {
     sFileSystem = fileSystem;
+    sInitialized = true;
```

Comment:
is there any reason for this class to store the filesystem statically? Could we make the filesystem field non-static and use `FileSystem.Factory.get()` in the no-arg constructor, e.g.

```java
AbstractFileSystem() {
  this(FileSystem.Factory.get());
}
```